pkgs/by-name/op/opencore-amr/package.nix

```diff
@@ -13,7 +13,7 @@ stdenv.mkDerivation rec {
   };
 
   meta = {
-    homepage = "https://opencore-amr.sourceforge.io/";
+    homepage = "https://sourceforge.net/projects/opencore-amr/";
```

Comment:
I am going to accept this since the website returns a 302 to the sourceforge page